package.json

```diff
@@ -97,9 +97,9 @@
 		"test:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/nyc/**\" nyc",
 		"test:coverage": "c8 npm test",
 		"test:fromroot": "mocha \"packages/**/dist/test/**/*.spec.*js\" --exit",
-		"test:jest": "assign-test-ports && pnpm puppeteer browsers install chrome && pnpm -r --no-sort --stream --no-bail test:jest --color",
-		"test:jest:bail": "assign-test-ports && pnpm puppeteer browsers install chrome && pnpm -r --no-sort --stream test:jest",
-		"test:jest:report": "assign-test-ports && pnpm puppeteer browsers install chrome && pnpm -r --no-sort --stream --no-bail --workspace-concurrency=4 test:jest --ci",
+		"test:jest": "assign-test-ports && pnpm pnpm puppeteer browsers install chrome-headless-shell && pnpm -r --no-sort --stream --no-bail test:jest --color",
```

Comment:
extra `pnpm`?

Reply:
too funny - indeed

Reply:
I propose leaving it to get fix in and then I'll submit follow on.